### PR TITLE
Back button, like button, flag button for Image Details page

### PR DESCRIPTION
**Pull request by Raymond-Xia**
- Added a back button to the Image Details page which navigates back to timeline view
- Added a like button which just displays a toast
- Added a flag button which just displays a toast
- Button sizing/style is abysmal but good enough for demo right now xd
image image image

### CODE REVIEW

**Review by kcchik — APPROVED** [review]
yum

File: app/src/main/java/com/example/snaplapse/ImageDetailsFragment.kt [hunk 1]
```diff
@@ -36,14 +33,31 @@ class ImageDetailsFragment(var item: ItemsViewModel, private val mList: List<Ite
...
+        val text = view.findViewById<TextView>(R.id.textView)
         text.text = resources.getString(R.string.image_details).format(item.text)
         val img: ImageView = view.findViewById(R.id.imageView) as ImageView
         img.setImageResource(item.image)
         img.setOnTouchListener(OnSwipeTouchListener(activity, item, mList, view))
 
-        // Inflate the layout for this fragment
+        val likeButton: ImageButton = view.findViewById(R.id.like_button) as ImageButton
+        likeButton.setOnClickListener {
+            Toast.makeText(activity, "Image liked!", Toast.LENGTH_SHORT).show()
```
Comment (danielkim10) [inline app/src/main/java/com/example/snaplapse/ImageDetailsFragment.kt]: ree

**Review by danielkim10 — APPROVED** [review]

File: app/src/main/java/com/example/snaplapse/ImageDetailsFragment.kt (hunk 1 above)
Comment (Raymond-Xia) [reply app/src/main/java/com/example/snaplapse/ImageDetailsFragment.kt]: These r supposed to be placeholders but I'll add it lol

Comment (kcchik) [reply app/src/main/java/com/example/snaplapse/ImageDetailsFragment.kt]: we fix l8r

Comment (danielkim10) [reply app/src/main/java/com/example/snaplapse/ImageDetailsFragment.kt]: yea idgaf u can just merge

**Pull request merged**